lms/static/js/bookmarks/views/bookmarks_list_button.js

```diff
@@ -16,11 +16,20 @@
             },
 
             initialize: function () {
-                this.bookmarksListView = new BookmarksListView({
-                    collection: new BookmarksCollection(),
-                    loadingMessageView: new MessageView({el: $(this.loadingMessageElement)}),
-                    errorMessageView: new MessageView({el: $(this.errorMessageElement)})
-                });
+                var bookmarksCollection = new BookmarksCollection(
+                    {
+                        course_id: $('.courseware-results').data('courseId'),
+                        url: $(".courseware-bookmarks-button").data('bookmarksApiUrl')
```

Comment:
`courseId` is also used by courseware search. 
`bookmarksApiUrl` is used on multiple places in bookmarks.
`langCode` is only used in bookmarks_list.js

I am not sure if we can pass variables from server through `main.js`